apps/molgenis-components/src/client/client.ts

```diff
@@ -238,7 +238,8 @@ _schema {
 }}`;
 
 const graphqlURL = (schemaName: string) => {
-  return schemaName ? "/" + schemaName + "/graphql" : "graphql";
+  const graphqlURL = schemaName ? "/" + schemaName + "/graphql" : "graphql";
+  return graphqlURL;
```

Comment:
why the extra const ?

Reply:
ah, yes, I can changes this back. There used to be a debug message in the middle.